scrapy/downloadermiddlewares/ajaxcrawl.py

```diff
@@ -89,8 +88,7 @@ def _has_ajaxcrawlable_meta(text):
     if 'content' not in text:
         return False
 
-    text = _script_re.sub(u'', text)
-    text = _noscript_re.sub(u'', text)
-    text = html.remove_comments(html.replace_entities(text))
+    text = html.remove_tags_with_content(text, ('script', 'noscript'))
```

Comment:
20% slowdown doesn't look too bad. Could you please try it with some data of size 32KB (this is what AjaxCrawlMiddleware sends as `text`)?